integration/nwo/fabric/network/network.go

```diff
@@ -181,16 +181,23 @@ func (n *Network) GenerateArtifacts() {
 func (n *Network) Load() {
 }
 
-func (n *Network) Members() []grouper.Member {
+func (n *Network) Members() ([]grouper.Member, error) {
 	members := grouper.Members{}
 
-	if r := n.OrdererGroupRunner(); r != nil {
-		members = append(members, grouper.Member{Name: n.Prefix + ".orderers", Runner: r})
+	runner, err := n.OrdererGroupRunner()
+	if err != nil {
```

Comment:
If we use `Expect(err).NotTo(HaveOccurred(), "failed to get orderer group members`)`, we could avoid to return an error here. We do this everywhere in NWO.
What do you think? 
This would avoid also the changes to the Platform interface.

Reply:
I can do that, I wasn't sure whether it was better to surface the error back up to the main point of NWO to panic it there or not. I guess this call will actually do a panic as well. I could just do that at the point where it looks for the command and the command was not found. What do you think ?

Reply:
yep, that works as well 👍